ecosystem/indexer-grpc/indexer-grpc-utils/src/file_store_operator/gcs.rs

```diff
@@ -4,20 +4,27 @@
 use crate::{constants::BLOB_STORAGE_SIZE, file_store_operator::*, EncodedTransactionWithVersion};
 use cloud_storage::{Bucket, Object};
 use itertools::{any, Itertools};
+use std::env;
 
 const JSON_FILE_TYPE: &str = "application/json";
+const SERVICE_ACCOUNT_ENV_VAR: &str = "SERVICE_ACCOUNT";
 
 pub struct GcsFileStoreOperator {
     bucket_name: String,
     /// The timestamp of the latest metadata update; this is to avoid too frequent metadata update.
     latest_metadata_update_timestamp: Option<std::time::Instant>,
+
+    /// The timestamp of the latest verification metadata update; this is to avoid too frequent metadata update.
+    latest_verification_metadata_update_timestamp: Option<std::time::Instant>,
 }
 
 impl GcsFileStoreOperator {
-    pub fn new(bucket_name: String) -> Self {
+    pub fn new(bucket_name: String, service_account_path: String) -> Self {
+        env::set_var(SERVICE_ACCOUNT_ENV_VAR, service_account_path);
```

Comment:
where/how is this used?

Reply:
currently, we implicitly require `SERVICE_ACCOUNT_ENV_VAR` as env var for cloud storage. this is to surface it up in config to unify all configuration into one place.

Reply:
I see. So it's a requirement in GCP?

Reply:
yea, and it's the `cloud-storage` requirement. i didn't see it exposes other ways to config the identity.

Reply:
maybe leave some comment there then? it's quite hard to identity lol

Reply:
done